setup.py

```diff
@@ -196,6 +196,12 @@ def _setup_entry_points() -> Dict:
         ]
     )
 
+    # recipe_template entrypoint
+
+    entry_points["console_scripts"].append(
+        "sparseml.pytorch.recipe_template=sparseml.pytorch.recipe_template.cli:main"
```

Comment:
Is the cli file supposed to be included in this PR?

Reply:
There is some base functionality but the full-proof cli is in another diff